api/meetings/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.conf.urls import url
+
+from api.meetings import views
+
+app_name = 'osf'
+
+urlpatterns = [
+    url(r'^$', views.MeetingList.as_view(), name=views.MeetingList.view_name),
+    url(r'^(?P<meeting_id>\w+)/$', views.MeetingDetail.as_view(), name=views.MeetingDetail.view_name),
+    url(r'^(?P<meeting_id>\w+)/submissions/$', views.MeetingSubmissionList.as_view(), name=views.MeetingSubmissionList.view_name),
```

Comment:
Matt also pointed out these could be private endpoints since they'll probably be temporary.